core/lib/rom/attribute.rb

```diff
@@ -312,9 +290,9 @@ def meta(opts = nil)
     #
     # @api public
     def inspect
-      opts = options.reject { |k| k == :type }
+      opts = options.reject { |k| [:type, :name].include?(k) }
       meta_and_opts = meta.merge(opts).map { |k, v| "#{k}=#{v.inspect}" }
-      %(#<#{self.class}[#{type.name}] #{meta_and_opts.join(' ')}>)
+      %(#<#{self.class}[#{type.name}] name=#{name.inspect} #{meta_and_opts.join(' ')}>)
```

Comment:
I think it makes sense to keep giving a special treatment to `name` in the `#inspect` option.